x-pack/plugins/entity_manager/server/lib/entity_client.ts

```diff
@@ -221,21 +225,30 @@ export class EntityClient {
     sources,
     start,
     end,
+    sortBy,
     metadataFields = [],
     filters = [],
     limit = 10,
   }: {
     sources: EntitySource[];
     start: string;
     end: string;
+    sortBy?: SortBy;
     metadataFields?: string[];
     filters?: string[];
     limit?: number;
   }) {
     const entities = await Promise.all(
       sources.map(async (source) => {
-        const mandatoryFields = [source.timestamp_field, ...source.identity_fields];
+        const mandatoryFields = [
```

Comment:
It's a little odd that these are called mandatory but it uses optional config? 
I guess we mean that IF the config is set, **then** it is mandatory? It's fine

Reply:
correct, mandatory if set